### DISCUSSION TITLE
Several overlays programmatically closing

### DISCUSSION THREAD

**Discussion by oshumelka** [fancyapps/ui#455] [Q&A] [answered]
Hi! 
Please help to resolve such issue. I have several dialogs opened. How to close each opened dialog programmatically? Fancybox.close() closes all of them.

**Comment by fancyapps** [comment, marked as answer]
Hi,

`Fancybox.close(false);` would close only the top-most instance. It is actually the same as `Fancybox.getInstance().close()`.

Also, if you store somewhere references to each instance, you can close them individually:

```js
const fb1 = new Fancybox([{html : "One"}]);
const fb2 = new Fancybox([{html : "Two"}]);

fb1.close();
fb2.close();
```
